lib/loader/serialize/serial_type.cpp

```diff
@@ -55,6 +69,14 @@ Serializer::serializeRefType(const ValType &Type, ASTNodeAttr From,
       OutVec.push_back(static_cast<uint8_t>(Code));
     }
     return serializeHeapType(Type, From, OutVec);
+  case TypeCode::NullExternRef:

```

Comment:
As commented before, the following cases will always not be reached because the `getCode()` for `reftype` will always be `TypeCode::Ref` or `TypeCode::RefNull`.